packages/lodestar-cli/src/cmds/beacon/cmds/init/index.ts

```diff
@@ -13,8 +14,15 @@ export const description = "Initialize lodestar beacon node";
 export function builder(yargs: Argv<IBeaconArgs>): Argv<IBeaconArgs> {
   yargs.options(beaconInitOptions);
   const args = parseArgs(yargs) as IBeaconInitArgs;
-  if (args.templateConfigFile) {
-    yargs.config(readBeaconConfig(args.templateConfigFile));
+
+  if (args.chain && args.chain.genesisStateFile) {
+    fs.readFileSync(args.chain.genesisStateFile, "utf8");
+  }
+
+  const {templateConfigFile} = args;
+  if (templateConfigFile) {
+    fs.readFileSync(templateConfigFile, "utf8");
+    yargs.config(readBeaconConfig(templateConfigFile));
```

Comment:
Can you investigate why there's a try/catch around `readBeaconConfig`? Likely should be removed as a better solution than this.

Reply:
yep.  just took out the try/catch

Reply:
and took out my initial solution